docs/_articles/en/localdev/set-up-lwc-local-dev.md

```diff
@@ -10,9 +10,9 @@ To set up LWC Local Development, install the server, enable and authorize a Dev
 Run this command from a command-line interface.
 
 ```
-sfdx plugins:install @salesforce/lwc-dev-server
+sf plugins install @salesforce/lwc-dev-server
```

Comment:
@daphne-sfdc should local dev also be using sf?

Reply:
@AnanyaJha Yes, the change to `sf` here is correct.  The command `sf plugins install` is a general command used to install any plugin - it doesn't matter that the plugin being installed is local dev server.